standards/SPIFFE_Federation.md

```diff
@@ -307,6 +307,56 @@ As described in the [Trustworthiness of the Bundle Endpoint Server](#74-trustwor
 
 Finally, it should be noted that the "links" in this chain are formed by individual HTTPS request operations (against the various bundle endpoint servers in the chain), and these operations are likely occurring at different times. SPIFFE bundle endpoint clients should be logging these HTTPS request operations, and administrators should take care to preserve these logs for future forensic analysis if necessary.
 
+### 7.7 Processing claims asserted by foreign trust domains
+
+SPIFFE Federation allows workloads in one trust domain to authenticate SVIDs, and all the claims therein, issued by other trust domains. By an "authentic" claim, we mean specifically that it can be safely assumed that the foreign trust domain authority _asserted_ the claim. This is not the same thing as saying that the claim is _true_, or that it is _correctly interpreted_.
+
+Operators should use caution and good judgement in evaluating if and how they should process claims asserted in SVIDs from foreign trust domains. The recommended posture is not to process foreign claims that have not been evaluated accoring to the framework described here (i.e. default deny), then assert exceptions for specific claims from specific trust domains to be allowed if they pass muster.
```

Comment:
```suggestion
Operators should use caution and good judgement in evaluating if and how they should process claims asserted in SVIDs from foreign trust domains. The recommended posture is to process foreign claims only after they have been evaluated according to the framework described below; for example, start with a default deny policy, then define exceptions for specific claims from specific trust domains to be allowed if they pass muster.
```

Reply:
What does "pass muster" mean in this context?

Reply:
It means that, after being evaluated as described, they survive that scrutiny and the operator approves of processing them.

Maybe we just cut that phrase out

"The recommended posture is to process foreign claims only after they have been evaluated according to the framework described below; for example, start with a default deny policy, then define exceptions for specific claims from specific trust domains."

I guess it's clear from the first clause of the sentence what the criteria is for an exception to the default deny?